src/teacher/templates/teacher/includes/pagination.html

```diff
@@ -0,0 +1,19 @@
+<div class="pagination" style="padding: 2rem 0">
```

Comment:
Pagination is identical in students, groups and teachers' templates. Could be save only once in root templates/includes.

Reply:
Fixed.